src/core/reducers/errors.js

```diff
@@ -41,12 +44,23 @@ function getMessagesFromError(error) {
     });
 
     if (apiMessages.length) {
-      messages = apiMessages;
+      data = { ...data, messages: apiMessages };
     } else {
       log.warn('API error response did not contain any messages', error);
     }
+
+    if (
+      error.response.data.detail &&
+      // TODO: switch to checking for a code after
+      // https://github.com/mozilla/addons-server/issues/5122
+      error.response.data.detail === 'Signature has expired.'
```

Comment:
This should at least be in a `gettext` call so we can grep for it, but I would really like for us to translate this message.

Also it should be changed. At least "Your session has expired." instead of "signature" which is confusing and less common terminology.